db/migrate/20211217152414_normalize_specimen_role_and_date_produced_samples.rb

```diff
@@ -0,0 +1,27 @@
+class NormalizeSpecimenRoleAndDateProducedSamples < ActiveRecord::Migration
+  def up
+    Sample.find_each do |sample|
+      unless sample.specimen_role.blank?
+        specimen_role_id = sample.specimen_role[0].downcase
+        sample.specimen_role = (Sample.specimen_role_ids_list.include? specimen_role_id) ? specimen_role_id : sample.specimen_role
```

Comment:
I think we're relying too much on Models for these migrations, and [that's an issue](https://guides.rubyonrails.org/v3.2/migrations.html#using-models-in-your-migrations).

Specifically, this `Sample.specimen_role_ids_list` method may be renamed in the code after we merge this migration, so if we want to run the migration in an instance later - the migration will break.

For this case, I think I'd hard-code the list of known `specimen_role_ids` in the migration itself.